src/ui/login/login.js

```diff
@@ -1,73 +1,110 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
 console.debug('login.js');
 import thoughtStorage from 'storage/thought-storage';
 import googleDriveApi from 'api/google-drive-api';
 import siteGlobalLoadingBar from 'ui/spinner/site-global-loading-bar';
-import { gapiAuthorize } from 'api/google-login';
-import loginPageHTML from 'text!./login.html';
+import {gapiAuthorize} from 'api/google-login';
 import googleApiLoader from 'api/google-api-loader';
 import './slide-1.png';
 import './slide-2.png';
 import './slide-3.png';
 import './slide-4.png';
 import './login.css';
 
-let element;
+export class LoginPage extends Component {
+  state = {googleApiInit: false};
```

Comment:
I'd call it `isGoogleApiInitialized`.

`googleApiInit` sounds more like a function which would initialize the google api.

Reply:
done